config/config.go

```diff
@@ -36,9 +43,16 @@ func (s *Settings) Load(filename string) error {
 	if err := yaml.Unmarshal(content, &s.config); err != nil {
 		return err
 	}
+	for _, c := range s.config.Instances {
+		if c.Interval == 0 {
+			c.setInverval(DefaultInteval)
+		}
+	}
+
```

Comment:
this is for the default value, if someone didn't set interval in `config.yaml`. ping @delgod for details

Reply:
I think, it kind of useful to have kind of setter here, because in future, you will want to set default for some other parameters, like region and etc

Reply:
@arvenil we need a default value for backward compatibility with old config format

Reply:
I understand. I don't say we don't need default. I say we don't need unnecessary setter func. Just modify `Interval` directly. Check the code I wrote it sets DefaultInterval. No need also for pointers.